modules/ShipStation/Api/src/Model/Repo/Order.php

```diff
@@ -35,27 +35,25 @@ public function getOrdersFromRenewDate($dtOrderUpdateStart = '', $dtOrderUpdateE
                 ->setParameter('lastRenewDateEnd', $dtOrderUpdateEnd);
         return $objQueryBuilder->getResult();
     }
+
     /**
      * Add Tracking number to the order
      *
-     * @param $intOrderNumber integer
+     * @param $order \XLite\Model\Order
      * @param $intTrackingNumber integer
      */
-    public function addOrderTrackingNumber($intOrderNumber = 0, $intTrackingNumber = 0) 
+    public function addOrderTrackingNumber($order = null, $intTrackingNumber = 0)
     {
-        $version = \XLite\Module\ShipStation\Api\Main::getMajorVersion();
-        if (floatval($version) >= 5.4) {
```

Comment:
We used to check the version to decide how to write the OrderTrackingNumber to the database. For newer versions we included the timestamp `NOW()`.

Looking at the DB, we actually don't need to do this, because if it's left out, it will default to the current Unix timestamp. By removing that part, we can skip the version comparison.